index.html

```diff
@@ -208,14 +212,18 @@ <h1>Skills</h1>
 
     <img class="illustration3" src="media\illustration-contact-form.svg" alt="illustration">
     
+    <div class="desktop-img">
+    <img class="desk-illustr1" src="./media/illustration-1-contact-form-desktop.svg" alt="">
+    <img class="desk-illustr2" src="./media/illustration-2-contact-form-desktop@2x.png" alt="">
+    <img class="desk-illustr3" src="./media/contact form-illustratoin-3-desktop.svg" alt="">
+    
     <div class="contact-text">
-        
-        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
     </div>
 
     <form action="https://formspree.io/f/xdojpwrd" method="POST">
         
-        <fieldset class="contact_info">
+        <fieldset>
 
                 <input class="input1" maxlength="30" type="text" placeholder="Full name" id="name" value=""
```

Comment:
- [ ] I can see you implemented the requested change by the previous reviewer but there is a small thing that is missing you were almost there check on the image below
- 
![as](https://user-images.githubusercontent.com/60146030/220963987-5fa50503-d903-46e0-8245-2fb2df11dc4e.PNG)

consider aligning this text to the cards above it as requested above try to make them have the same margin you can do so by decreasing the font size you are giving the text and reduce the margin